src/components/TopBar/AboutDialog.js

```diff
@@ -0,0 +1,364 @@
+                                whiteSpace: 'pre',
+                            },
+                            '&:first-child': {
+                                '&:before': {
+                                    content: "''",
+                                },
+                            },
+                        },
+                        dd: {
+                            fontStyle: 'italic',
+                        },
+                    }}
+                >
+                    {appLicense && (
+                        <>
+                            <dt>
+                                <FormattedMessage id="about-dialog/license" />
+                            </dt>
+                            <dd>{appLicense}</dd>
+                        </>
+                    )}
+                    <dt>Version</dt>
+                    <dd>{appVersion || '?unknown?'}</dd>
+                </Box>
+                <Box component="p">
+                    GridSuite version{' '}
+                    {!loadingGlobalVersion && <b>{actualGlobalVersion || '?unknown?'}</b>}
+                    <Fade
+                        in={loadingGlobalVersion}
+                        style={{ transitionDelay: '500ms' }}
+                        unmountOnExit
+                    >
+                        <CircularProgress size="1rem" />
+                    </Fade>
+                </Box>
+                {actualGlobalVersion !== null &&
+                    startingGlobalVersion !== actualGlobalVersion && (
+                        <Collapse in={open}>
+                            <Alert severity="warning" variant="outlined">
+                                <AlertTitle>
+                                    <FormattedMessage id="about-dialog/alert-running-old-version-title" />
+                                </AlertTitle>
+                                <FormattedMessage id="about-dialog/alert-running-old-version-content" />
+                            </Alert>
+                        </Collapse>
+                    )}
+                {getAdditionalComponents && (
+                    <Box
+                        sx={{
+                            '.MuiPaper-root': {
+                                width: '100%',
+                            },
+                            '.MuiCard-root': {
+                                height: '100%',
+                                width: '100%',
+                            },
+                            '.MuiCardHeader-root': {
+                                padding: 1,
+                            },
+                            '.MuiCardContent-root': {
+                                padding: 1,
+                                paddingTop: 0,
+                            },
+                        }}
+                    >
+                        <Box component="p">
+                            <FormattedMessage id="about-dialog/components-version" />
```

Comment:
Insteas of "Components", I would use "Applications" or "Services", it's more explicit for the user.

Reply:
hum... yes I haven't found which terme use: we have application and servers here, with optionally other stuff passed by the front

Reply:
the PO have an idea but the designer found: "modules" 👍🏻